tests/test_pe_reports.py

```diff
@@ -0,0 +1,33 @@
+"""File to be used to create test for pe-reports."""
+
+# Standard Python Libraries
+import unittest
+
+
+class TestingStringMethods(unittest.TestCase):
+    """Simple test for first-commit only."""
+
+    def test_string_equality(self):
+        """Calculates the string output."""
+        # if both arguments are equal then it's success
+        self.assertEqual("ttp" * 5, "ttpttpttpttpttp")
+
+    def test_string_case(self):
+        """Compare two strings."""
+        # if both arguments are equal then it's success
+        self.assertEqual("tutorialspoint".upper(), "TUTORIALSPOINT")
+
+    # checking whether a string is upper or not
```

Comment:
This comment can be removed now that we have docstrings.

Reply:
Resolved in 8806233